addons/website_sale/views/templates.xml

```diff
@@ -490,8 +490,10 @@
     <!-- OffCanvas Nav -->
     <template id="website_sale.o_wsale_offcanvas" name="Offcanvas">
         <aside id="o_wsale_offcanvas"
-               class="offcanvas offcanvas-end p-0">
-
+               class="o_website_offcanvas offcanvas offcanvas-end p-0">
+            <div class="offcanvas-header justify-content-end">
+                <button type="button" class="btn-close" data-bs-dismiss="offcanvas" aria-label="Close"/>
```

Comment:
`data-bs-dismiss="offcanvas"` -> Since we may have header's offcanvas too, better to define an unique ID for the website_sale one, something like `o_wsale_offcanvas`